src/components/Dashboard/Dashboard.module.scss

```diff
@@ -146,16 +132,45 @@
   }
 }
 
+.inputWrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  width: 100%;
+
+  @media (min-width: $screen-md-min) {
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 1rem;
+  }
+
+  label {
+    width: 50%;
+    margin-top: 0.7rem;
+  }
+}
+
 .input {
   margin-left: 1rem;
-  width: max-content;
+  width: min-content;
   color: $translucent-silk;
   background-color: $hornblende-green;
   border: none;
   border-bottom: 1px solid $translucent-silk;
```

Comment:
Do we suddenly have a new input type that is one-off? What a wild design! Just remember, if you have a considerable project and create a random design for every form, it will be unmaintainable in literally like four forms :D.

But sure, if you want it, go for it. If you would use this thing somewhere else, we should introduce a "variant" for the input so the styling has a single source of truth and is not all over the place.

Reply:
Wild design was not the plan, but it came with the proces. Fixed it with two variants :) but since it was change in text field component i had to modify more files.